### PR TITLE
Web development - added a youtube channel link

### PR DESCRIPTION
**Pull request by RitaDee**
**IMPORTANT:** When creating your Pull Request, please use the following format for the title: `Category - Resource Name & Section`  
Category - Web development

Please follow these steps to create the PR title correctly:

> - Replace "Category" with the relevant category, such as **Web Development** or **UI & UX Design**.
> - Replace "Resource Name & Section" with the specific name of the resource and the relevant section where it belongs. For instance, you could use something like **Web development - CSS Portal & Tools** or **Web development - Leetcode & Challenges**.

### Fixes Issue

https://www.youtube.com/watch?v=bMknfKXIFA8

### Description: 

Added a YouTube channel link (Web development) 

### Resource URL: 

[https://www.youtube.com/watch?v=bMknfKXIFA8]

### Checklist:
Before submitting your pull request, ensure that you have completed the following tasks: 

- [x] I have carefully reviewed and adhered to the [contributing guidelines](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md) before creating this pull request.
- [x] I followed the prescribed PR title template. _(Check this if you're adding a resource)_

### CODE REVIEW

**Comment by RitaDee** [comment]
@lr2120, please can you help me with this PR?

File: Web Development/README.md [hunk 1]
```diff
@@ -348,6 +348,11 @@ to everyone - everywhere!</td>
         <td>Hitesh Choudhari is a YouTube channel that offers various courses on programming languages. It is one of the best channel to learn web development at free of cost.
         </td>
       </tr>
+      <tr>
+        <td><a href="">https://www.youtube.com/watch?v=bMknfKXIFA8</a>FreeCodeCamp</td>
```
Comment (jfmartinz) [inline Web Development/README.md:352]: Hello @RitaDee It looks good but the problem is that freeCodeCamp youtube channel exist already, change it to something else and then I will merged once youre done.

```suggestion
        <td><a href="">https://www.youtube.com/watch?v=bMknfKXIFA8</a>FreeCodeCamp</td>
```

File: Web Development/README.md [hunk 2]
```diff
@@ -348,6 +348,11 @@ to everyone - everywhere!</td>
...
+        <td>React course for beginners. It simplifies and dymystifies react to any beginner who is willing and eager to learn.
+        </td>
+      </tr>
```
Comment (jfmartinz) [inline Web Development/README.md:355]: Hello @RitaDee It looks good to me but the problem is that this yt link you added is suitable for `Turorial or Courses` section. Now please can you move it to that section and then I will merged once you're finished. Thank you!

File: Web Development/README.md (hunk 1 above)
Comment (jfmartinz) [inline Web Development/README.md:352]: Another problem is this. Please change this code to this one:
`<td><a href="https://www.youtube.com/watch?v=bMknfKXIFA8">React Course - Beginner's Tutorial for React JavaScript Library [2022]</a></td>`

Comment (RitaDee) [reply Web Development/README.md]: Done

File: Web Development/README.md (hunk 2 above)
Comment (RitaDee) [reply Web Development/README.md]: Done. Thank you

**Comment by lr2120** [comment]
Hi, what kind of help do you need specifically. Would you like me to write you a description for this video. How about this?

Dive into this comprehensive React tutorial by FreeCodeCamp.org, a renowned platform offering a multitude of free resources for aspiring developers.

**Pull request merged**